aspects/electrical/PowerBus/test/UtGunnsElectDistributed2WayBus.cpp

```diff
@@ -311,16 +311,10 @@ void UtGunnsElectDistributed2WayBus::testUpdateForcedRole()
     UT_RESULT;
 
     /// - Initialize default constructed test article with nominal initialization data.
-    GunnsElectDistributed2WayBusSupplyData* supply1 = tArticle->createSupplyData();
+    GunnsElectDistributed2WayBusSupplyData* supply1 __attribute__((unused)) = tArticle->createSupplyData();
```

Comment:
This line should just be deleted, since supply1 never got used.  This might be causing the CI job failure in the PowerBus folder in the ASan unit test - i'm not sure why but maybe Address Sanitizer didn't care about the unused variable until you added __attribute__((unused)) to it... Go ahead and just delete this line and resubmit.